Bio/EBI/ebisearch.py

```diff
@@ -0,0 +1,751 @@
+# Copyright 2017 by Berenice Batut (berenice.batut@gmail.com). All rights reserved.
+# Revision copyright 2017 by Francesco Gastaldello. All rights reserved.
+#
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+
+"""Provide code to access the EBI Search API.
+
+This module aims to make EBI Search API easy to use. See:
+https://www.ebi.ac.uk/ebisearch/overview.ebi
+
+The EBI Search API is a text search engine to access biological data resources
+hosted at the European Bioinformatics Institute (EMBL-EBI).
+
+The following code was developed by Berenice Batut (berenice.batut@gmail.com)
+
+References:
+Park Y.M., Squizzato S., Buso N., Gur T., Lopez R. (2017)
+The EBI search engine: EBI search as a service making biological data accessible
+for all.;Nucleic Acids Research, May 2, 2017; DOI: 10.1093/nar/gkx359
+
+"""
+
+import requests
+
+import requires_internet
```

Comment:
We only use this in our tests framework, so you can remove it from the main module.